assets/js/custom/CustomTranslation.js

```diff
@@ -0,0 +1,28 @@
+import $ from 'jquery'
+import Swal from 'sweetalert2'
+
+export class CustomTranslation {
```

Comment:
not quite what i thought. It's simpler than this, like change the name of this class to `closeEditorDialog`, and just keep this one function in there, maybe rename it to `show`, so it will be used like `closeEditDialog.show()`

Then other stuff like snackbar would like in it's own class and look like `snackbar.show()`
api might look like `api.saveCustomDescription(language, message)` and `api.saveCustomCredit(...)`

What do you think?

Reply:
I like it, you would still need to create an instance of the class though right?

Reply:
yes, inject the dependencies in the ctor

Reply:
Sounds good. I added a few files for the popups, snackbar, and API. Is there anything else you think I should do?